### PR TITLE
EI heat process

### PR DESCRIPTION
**Pull request by ccomb**
[story](https://www.notion.so/Int-gration-Ecoinvent-Mix-Chaleur-f250a03d16754bd2b191bc3738f1b4cc)

### CODE REVIEW

File: public/data/textile/processes.json [hunk 1]
```diff
@@ -8458,7 +8458,7 @@
   },
   {
     "search": "",
-    "name": "(EI) Heat, natural gas, France",
+    "name": "(EI) Heat mix (FR)",
```
Comment (n1k0) [inline public/data/textile/processes.json:8461]: As the impacts didn't change, does that mean the previous label was wrong?

File: src/Data/Textile/Inputs.elm [hunk 2]
```diff
@@ -139,7 +137,6 @@ type alias Query =
     , fading : Maybe Bool
     , dyeingMedium : Maybe DyeingMedium
     , printing : Maybe Printing
-    , ennoblingHeatSource : Maybe HeatSource
```
Comment (n1k0) [inline src/Data/Textile/Inputs.elm:142]: This is a breaking change for the API, so we must add an entry to the Api changelog. Also, we should update the `openapi.yml` docs accordingly.

File: public/data/textile/processes.json (hunk 1 above)
Comment (n1k0) [reply public/data/textile/processes.json:8461]: Scratch this, code changed meanwhile.

File: src/Data/Textile/Inputs.elm (hunk 2 above)
Comment (ccomb) [reply src/Data/Textile/Inputs.elm:142]: Sure, thx for noticing

**Review by n1k0 — APPROVED** [review]
LGTM 👍

**Pull request merged**